### ISSUE TITLE
Accessibility], WCAG 1.4.3. The contrast of the links in the News section is too low

### ISSUE THREAD

**Issue by Eiskore** [NatLibFi/Skosmos#1963]
## URL address of the page where you encountered the problem
https://test.dev.finto.fi/fi/ 

## Description of the problem
The News section has bright blue links. Links have insufficient color contrast of 3.55 (foreground color: #0d6efd, background color: #dfe5ed). Expected contrast ratio is at least  4.5:1

Fix: Use darker foreground color for link, eg.  #0a5dd8 and the original background color: #dfe5ed to meet a contrast ratio of 4.64:1

Severity: Low
Priority: Medium

## Additional information (e.g. screenshots) about the problem

<img width="714" height="294" alt="Image" src="https://github.com/user-attachments/assets/d4c9a0c7-470f-494d-b11e-7576e94e8a25" />

<img width="619" height="108" alt="Image" src="https://github.com/user-attachments/assets/4d215a8f-74b2-46d8-adea-5a888eb0183a" />

<img width="1004" height="572" alt="Image" src="https://github.com/user-attachments/assets/0c0113c3-49cd-43f2-a6c8-a639d201267e" />

## The browser you used when the problem appeared
Chrome, Firefox, Edge

**Comment by osma** [comment]
Fixed with the news section redesign in this Finto-data commit: https://github.com/NatLibFi/Finto-data/commit/4a18be6c0b9c0ef70566f6a077f784a3a84ce1bf

**Comment by tinskuu** [comment]
This issue is fixed.
